students/8/main.js

```diff
@@ -53,10 +53,44 @@ const ex9 = () => {
     console.log(calculateTotal(itemArray))
 }
 
+const ex10 = () => {
+    let usageData = [
+        { userId: 1111, minutes: 10.0 },
+        { userId: 2222, minutes: 20.0 },
+        { userId: 3333, minutes: 30.0 }
+    ]
+
+    let person = { userId: 3333, fullName: "Charlie" }
+    let result = calcCellBillForPerson(usageData, person)
+    console.log(result)
+}
+
 //
 // Your functions here...
 //
 
+//Ex10
+
+const calcCellBillForPerson = (usageData, person) => {
+    let personBill = {}
+    let personCharge = 0
+    let MinuteCharge = { PerMinuteCharge: 0.1 }
```

Comment:
Let's change the case of this to `minuteCharge`.  Does it make sense to make this a `const`?

Reply:
Yes it does and I've made that change.